source/developer/localization.rst

```diff
@@ -28,9 +28,9 @@ To translate a language:
 Translations Updates
 --------------------
 
-1 - New and updated strings are imported to the `Mattermost Translation Server <http://translate.mattermost.com>`__ each day at 00:00 GMT. Strings with a ``mobile`` prefix are imported from the `mattermost-mobile <https://github.com/mattermost/mattermost-mobile>`__ project, others are from `mattermost-server <https://github.com/mattermost/mattermost-server>`__ and `mattermost-webapp <https://github.com/mattermost/mattermost-webapp>`__.
+1 - New and updated source strings are being automatically added to the translation server on GitHub pushes by developers.
```

Comment:
@metanerd To confirm, do you mean new and updated source strings are automatically added to the translation service each time a relevant GitHub PR is merged?

Reply:
Yes, I do mean exactly that.

Reply:
Should I use your phrasing?

Reply:
@metanerd yes, lets